vggish_audio_segmenter.py

```diff
@@ -18,14 +18,43 @@ def __init__(self, embedding_dim: int = 128, *args, **kwargs ):
         super().__init__(*args, **kwargs)
         self._dim = embedding_dim
 
+        def _segment(self):
+            pass # to do
+
     @requests
     def segment(self, docs: Optional[DocumentArray], **kwargs):
         """
-        Encode all docs with images and store the encodings in the embedding attribute of the docs.
-        :param docs: documents sent to the encoder. The docs must have `blob` of the shape `256`.
+        Encode all docs with audio and store the segmented regions in the attribute of the docs.
+        :param docs: documents sent to the segmenter. The docs must have `blob` of the shape `256`.
         """
         if not docs:
             return
+        docs = self._get_input_data(docs)
```

Comment:
do not overwrite the input docs. Bad pattern

Reply:
changed